sale_packaging_price/models/sale_stock.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, fields, api
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    packaging_price = fields.Float(
+        related='product_packaging.list_price', string='Package Price',
+        readonly=True)
+    product_packaging = fields.Many2one(ondelete='restrict')
```

Comment:
You have to add a domain to restrict package selection to those that the product allows.